content/blog/2022-03-29-march-22-community-gems.md

```diff
@@ -0,0 +1,98 @@
+---
+title: March '22 Community Gems
+date: 2022-03-29
+description: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+picture: 2022-03-29/mar-comm-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/march-22-community-gems/1119
+tags:
+  - Data Versioning
+  - DVC Remotes
+  - DVC API
+  - DVC Stages
+  - Community
+---
+
+### [What is the difference between using `dvc exp` run and `dvc repro`?](https://discord.com/channels/485586884165107732/485596304961962003/939070512322195456)
+
+This is a really good question from @v2.03.99!
+
+When you use `dvc exp run`, DVC automatically tracks each experiment run. Using
+`dvc repro` leaves it to the user to track each experiment.
+
+### [What is a good way to debug DVC stages in VSCode?](https://discord.com/channels/485586884165107732/485596304961962003/939269709780643861)
+
+A great question here from @quarkquark!
+
+You can debug in VSCode by following the steps below:
+
+- Install the `debugpy` package.
+- Navigate to `"Run and Debug" > "Remote Attach" > localhost > someport`.
+- In a terminal in VSCode,
+  `python -m debugpy --listen someport --wait-for-client -m dvc mycommand`
+
+This should help you debug the stages in your pipeline in the IDE.
```

Comment:
Link for more info: https://github.com/iterative/dvc/wiki/Debugging-DVC-interactively.